devel/hub/Portfile

```diff
@@ -39,18 +40,18 @@ post-extract {
 }
 
 destroot {
-    xinstall -W ${worksrcpath} bin/hub ${destroot}${prefix}/bin
+    xinstall -m 0755 -W ${worksrcpath} bin/hub ${destroot}${prefix}/bin
     set man-pages ${destroot}${prefix}/share/man/man1
     xinstall -d ${man-pages}
     foreach m [glob -directory ${worksrcpath}/share/man/man1 *.1] {
-        xinstall -W ${worksrcpath} ${m} ${man-pages}/
+        xinstall -m 0444 -W ${worksrcpath} ${m} ${man-pages}/
```

Comment:
These should probably be 0644?

Reply:
@pmetzger Seems both 644 and 444 are okay. Have a check of files in `/usr/share/man/man*`. I remember I firstly set it 0644 and changed it to 0444 after seeing 0444 used in another Portfile.

Reply:
It really should be 644 on any normal system. The other portfile was wrong.

Reply:
@pmetzger Okay, fixed.